common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -184,7 +184,6 @@ def expected_settings_names(self):
             'no_grade',
             'display_coursenumber',
             'display_organization',
-            'end_of_course_survey_url',
```

Comment:
This is the list of settings that are expected to be displayed in Advanced Settings (bok choy test).

There is not currently a bok choy test verifying the deprecated settings. I did not think it was worthwhile to add one at this time.